pkg/systemd/overview-cards/realmd.jsx

```diff
@@ -187,6 +187,12 @@ export class RealmdClient {
         if (cockpit.transport.host !== "localhost")
             return Promise.resolve();
 
+        const server_sw = find_detail(realm, "server-software");
+        if (server_sw !== "ipa") {
+            console.log("cleaning up ws credentials not supported for server software", server_sw);
```

Comment:
Wonder if this should be a `console.error`, but LGTM.

Reply:
No, it's not an error, it's expected. If anything, it should be console.debug(), but that's by and large the same as .log() in practice.